### PR TITLE
[Windows] Create multiple windows

### PR DESCRIPTION
**Pull request by loic-sharma**
Update the Windows runner to create multiple windows.

image

### CODE REVIEW

File: windows/runner/flutter_window.cpp [hunk 1]
```diff
@@ -6,10 +6,8 @@
 
 #include <optional>
 
-#include "flutter/generated_plugin_registrant.h"
-
-FlutterWindow::FlutterWindow(const flutter::DartProject& project)
-    : project_(project) {}
+FlutterWindow::FlutterWindow(std::shared_ptr<flutter::FlutterEngine> engine)
+    : engine_(std::move(engine)) {}
```
Comment (loic-sharma) [inline windows/runner/flutter_window.cpp:10]: TODO: Update the README's engine commit ID once https://github.com/loic-sharma/flutter-engine/pull/46 is merged

Comment (goderbauer) [reply windows/runner/flutter_window.cpp:10]: Unrelated: Any chance as part of that we could also merge in the latest changes from upstream?

Comment (loic-sharma) [reply windows/runner/flutter_window.cpp:10]: Yup I can do that after the Windows prototype lands (I'm expecting conflicts in the Windows embedder)

**Review by goderbauer — APPROVED** [review]
LGTM!

**Pull request merged**